<?xml version="1.0"?>

<!-- map.launch -->
<launch>

    <node name="generate_path_server" pkg="igvc_navigation" type="generate_path_server" output="screen" >
        <param name="goal_threshold" type="double" value="0.8"/>
        <param name="c_space" type="double" value="0.5"/>
        <param name="probability_threshold" type="double" value="0.7"/>
        <param name="point_turns_enabled" type="bool" value="false"/>
        <param name="reverse_enabled" type="bool" value="false"/>
        <param name="max_jump_size" type="double" value="10.0"/>
        <param name="theta_filter" type="double" value="1.5"/>
        <param name="max_theta_change" type="double" value="0.8"/>
        <param name="theta_change_window" type="double" value="3.0"/>
    </node>


</launch>
